src/coq_rules.ml

```diff
@@ -8,6 +8,27 @@ module SC = Super_context
 
 let coq_debug = false
 
+(* Coqdep / Coq expect the deps to the directory where the plugin cmxs
+   file are. This seems to correspond to src_dir. *)
+module Util = struct
+
+  let to_iflags dirs =
+    Arg_spec.S
+      (Path.Set.fold dirs ~init:[] ~f:(fun dir acc ->
+         Arg_spec.Path dir :: A "-I" :: acc)
+       |> List.rev)
+
+  let include_paths ts ~stdlib_dir =
+    let dirs =
+      List.fold_left ts ~init:Path.Set.empty ~f:(fun acc t ->
+        Path.Set.add acc (Lib.src_dir t))
+    in
+    Path.Set.remove dirs stdlib_dir
```

Comment:
Why do you remove the `stdlib_dir`? this seems like some OCaml like hack.

Reply:
Umm, good question; I do as in done in `Lib.include_paths`, but I am not sure why is needed, so I have removed this hack for now.